老國音.js

```diff
@@ -0,0 +1,291 @@
+/* 推導老國音
+
+音系：https://zh.wikiversity.org/zh-hant/%E8%80%81%E5%9C%8B%E9%9F%B3%E9%9F%B3%E7%B4%A0
+音變規則：https://zh.wikiversity.org/zh-hant/%E8%80%81%E5%9C%8B%E9%9F%B3%E8%88%87%E5%BB%A3%E9%9F%BB%E5%B0%8D%E6%AF%94
+字音來源：https://github.com/baopaau/rime-bepemefeve/blob/main/bepemefeve.dict.yaml
+注意「僞老國音」：https://www.bilibili.com/read/cv17377530/
+
+@author 307587 */
+
+/** @type { 音韻地位['屬於'] } */
+const is = (...x) => 音韻地位.屬於(...x);
+/** @type { 音韻地位['判斷'] } */
+const when = (...x) => 音韻地位.判斷(...x);
+
+if (!音韻地位) return [
+  ['注音符號\n否的話就用國際音標', true],
+  ['ㄬㄭ|ㄬ、ㄭ音標\n啟用「注音符號」時無效', [2, 'ȵ、ɿ、ʅ', 'ɲ、ɹ̩、ɻ̍']],
+  ['介母', [1,
+    { text: '元音：i u y', value: '元音' },
+    { text: '半元音：j w ɥ', value: '半元音' },
+  ], { hidden: 選項.注音符號 === true }],
+  ['ㄚ|ㄚ元音\nㄧㄞ：寬式/iai/、嚴式[iæi]\nㄧ、ㄩ後的ㄢ：寬式/-an/、嚴式[-ɛn]', [2, '嚴式', '寬式']],
+  ['ㄨㄣ|舌冠聲母後的ㄨㄣ讀音\n趙元任《國音新詩韻》：「ㄨㄣ」也可以一律照樣拼念作「ㄨㄜㄋ」，但是在「ㄉ，ㄊ，ㄋ，ㄌ；ㄗ，ㄘ，ㄙ」七個聲母後頭可以省去「ㄜ」音念「ㄨㄋ」', [2, 'un', 'uən']],
+  ['喉塞|標示入聲喉塞音韻尾', true, { hidden: 選項.注音符號 !== false }],
+  ['調式', ['調符',
+    '調值（趙元任）', '調值（王璞）', '調線（趙元任）', '調線（王璞）',
+    ...(選項.注音符號 === false ? ['附標'] : []), '調號上標', '調號', '調符',
+  ]],
+  ['聲調在前\n調式爲「附標」時無效', false],
```

Comment:
```suggestion
  ['聲調', [1, '五度符號（趙元任）', '五度符號（王璞）', '調值數字（趙元任）', '調值數字（王璞）', '調類數字', '調類數字（不上標）'], { hidden: 選項.注音符號 !== false }],
```

可以考虑这样。注音符号的调号不能用在国际音标上。调号标在前的作用是什么？另外可以考虑对注音符号加一个“省略陰平調號”的选项

Reply:
>注音符号的调号不能用在国际音标上

那是[組合附加符號](https://zh.wikipedia.org/zh-hk/%E7%B5%84%E5%90%88%E9%99%84%E5%8A%A0%E7%AC%A6%E8%99%9F)，目的就是讓你與其他字元自由搭配
>调号标在前

是參考「[寬式國際音標（`wong_ipa`）](https://github.com/CanCLID/rime-cantonese-schemes-editor?tab=readme-ov-file)」的標法

Reply:
国际音标里 ā 是中平调，á 是高平调，ǎ 是升调，à 是低平调，所以这套附加符号不能直接搬进国际音标；现代汉语研究也几乎没人在写国际音标时用附加符号表示声调，所以就不用考虑这个选项啦。调形标在音节前的写法在 1989 年就作废了，现在国内外学界没有人用，所以也不用考虑啦